### PR TITLE
Add textproto generator for bpconsume CLI

### PR DESCRIPTION
**Pull request by ayushmjain**

### CODE REVIEW

File: cli/bpconsume/jumpstartsolutions/solutionprotogenerator.go [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+	}
+}
+
+// addDeploymentTimeEstimate adds the deployment time for the solution to the
+// solution object from the BlueprintMetadata object.
+func addDeploymentTimeEstimate(solution *gen_protos.Solution, bpObj *bpmetadata.BlueprintMetadata) {
+	if bpObj.Spec.DeploymentDuration.ConfigurationSecs > 0 && bpObj.Spec.DeploymentDuration.DeploymentSecs > 0 {
+		solution.DeploymentEstimate = &gen_protos.DeploymentEstimate{
+			ConfigurationMinutes: int32(bpObj.Spec.DeploymentDuration.ConfigurationSecs / 60),
+			DeploymentMinutes:    int32(bpObj.Spec.DeploymentDuration.DeploymentSecs / 60),
```
Comment (prateekj0394) [inline cli/bpconsume/jumpstartsolutions/solutionprotogenerator.go:49]: We should ceil this value. otherwise estimate will be off by lower minute value

**Review by prateekj0394 — CHANGES_REQUESTED** [review]
Fix the estimate calculation logic

File: cli/bpconsume/jumpstartsolutions/solutionprotogenerator.go (hunk 1 above)
Comment (ayushmjain) [reply cli/bpconsume/jumpstartsolutions/solutionprotogenerator.go]: Done.

**Review by prateekj0394 — APPROVED** [review]
LGTM

**Pull request merged**